botfront/imports/ui/components/index.jsx

```diff
@@ -22,7 +22,7 @@ class Index extends React.Component {
         if (Meteor.userId()) {
             Tracker.autorun(() => {
                 if (Meteor.user() && areScopeReady() && projectsReady) {
-                    const projectIds = getScopesForUser(Meteor.userId(), 'owner');
+                    const projectIds = getScopesForUser(Meteor.userId(), 'owner') || Projects.find({}, { fields: {} }).fetch().map(project => project._id);
```

Comment:
I believe, it's a "fallback" so it may rarely get called and won't cause any issues in the day to day usage. But if the fallack is needed it won't work. Moreover the {fields: {}}  purpose is not clear. So I think we should remove it